doc/source/data/examples/nyc_taxi_basic_processing.ipynb

```diff
@@ -316,7 +316,7 @@
       "id": "a3fb551b",
       "metadata": {},
       "source": [
-       "To get a better sense of the data size, we can calculate the size in bytes of the full dataset. Note that for Parquet files, this size-in-bytes will be pulled from the Parquet metadata (not triggering a data read) and will therefore be the on-disk size of the data; this might be significantly smaller than the in-memory size!"
```

Comment:
removing `be the on-disk size of the data`, because precisely the size-in-bytes is the Parquet in-memory encoded data size, not Parquet on-disk compressed data size.